<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>130</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Folio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Nombre_p">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>180</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>311</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/logoN.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>290</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vacuna</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Folio_apli">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>130</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>241</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic Medium</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Comprobante</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbInfo_apli">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>120</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}


QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}</string>
    </property>
    <property name="text">
     <string>CONSULTAR</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMENU_paci2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>490</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	background-color: rgba(255, 255, 255, 5);
	border-radius:10px;
	border:10px;
}


QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}</string>
    </property>
    <property name="text">
     <string>MENU</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../imagenes/icono_home.png</normaloff>../imagenes/icono_home.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>480</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../imagenes/rosa.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbImprimir_apli">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>486</y>
      <width>151</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}


QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}</string>
    </property>
    <property name="text">
     <string>IMPRIMIR</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>340</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dosis</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_dosis_p">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>330</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_vacunaN">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>280</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>390</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fecha de aplicaci√≥n</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_fechaA">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>380</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_edadp">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
border: .5px solid
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Edad</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLimpiar">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>486</y>
      <width>151</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Dubai Medium</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color:rgb(205, 187, 227);
	border-radius:10px;
	border:10px;
}


QPushButton:hover{
background-color: rgb(170, 255, 127);
	border-radius:10px;
	border: 10px;
}</string>
    </property>
    <property name="text">
     <string>LIMPIAR</string>
    </property>
   </widget>
   <zorder>label_17</zorder>
   <zorder>label_3</zorder>
   <zorder>label_9</zorder>
   <zorder>lineEdit_Nombre_p</zorder>
   <zorder>label_2</zorder>
   <zorder>label_12</zorder>
   <zorder>lineEdit_Folio_apli</zorder>
   <zorder>label</zorder>
   <zorder>pbInfo_apli</zorder>
   <zorder>pbMENU_paci2</zorder>
   <zorder>pbImprimir_apli</zorder>
   <zorder>label_14</zorder>
   <zorder>lineEdit_dosis_p</zorder>
   <zorder>lineEdit_vacunaN</zorder>
   <zorder>label_15</zorder>
   <zorder>lineEdit_fechaA</zorder>
   <zorder>lineEdit_edadp</zorder>
   <zorder>label_4</zorder>
   <zorder>pbLimpiar</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
